Source/CronosPlayUnreal/CronosPlayUnreal.Build.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.IO;
+using UnrealBuildTool;
+
+public class CronosPlayUnreal : ModuleRules {
+
+  protected string ThirdPartyPath {
+    get {
+      return Path.GetFullPath(
+          Path.Combine(ModuleDirectory, "..", "..", "ThirdParty"));
+    }
+  }
+
+  public CronosPlayUnreal(ReadOnlyTargetRules Target) : base(Target) {
+    PCHUsage = ModuleRules.PCHUsageMode.UseExplicitOrSharedPCHs;
+    CppStandard = CppStandardVersion.Cpp17;
+
+    string LibrariesPath = Path.Combine(ThirdPartyPath);
+
+    PublicIncludePaths.AddRange(new string[] {});
+
+    PrivateIncludePaths.AddRange(new string[] { Path.Combine(ThirdPartyPath) });
+
+    PublicDependencyModuleNames.AddRange(
+        new string[] { "Core", "Json", "JsonUtilities", "HTTP"
+
+        });
+
+    PrivateDependencyModuleNames.AddRange(new string[] {
+      "CoreUObject",
+      "Engine",
+      "Slate",
+      "SlateCore",
+    });
+
+    DynamicallyLoadedModuleNames.AddRange(new string[] {});
+
+    if (UnrealTargetPlatform.Win64 == Target.Platform) {
+      PublicAdditionalLibraries.Add(Path.Combine(
+          LibrariesPath, "cronosplay", "lib", "Win64", "play_cpp_sdk.lib"));
+
+      PublicAdditionalLibraries.Add("Ncrypt.lib");
+      PublicAdditionalLibraries.Add("userenv.lib");
+      PublicAdditionalLibraries.Add("bcrypt.lib");
+      PublicAdditionalLibraries.Add("ntdll.lib");
+      PublicAdditionalLibraries.Add("Secur32.lib");
+      PublicAdditionalLibraries.Add("crypt32.lib");
+      PublicAdditionalLibraries.Add("ws2_32.lib");
```

Comment:
these exists on the end platform?

Reply:
yes, it's default library from visual studio